cmd/cli/mesh_list.go

```diff
@@ -72,11 +72,24 @@ func (l *meshListCmd) run() error {
 	return nil
 }
 
-func (l *meshListCmd) selectMeshes() (*v1.DeploymentList, error) {
-	deploymentsClient := l.clientSet.AppsV1().Deployments("") // Get deployments from all namespaces
+// getControllerDeployments returns a list of Deployments corresponding to osm-controller
+func getControllerDeployments(clientSet kubernetes.Interface) (*v1.DeploymentList, error) {
+	deploymentsClient := clientSet.AppsV1().Deployments("") // Get deployments from all namespaces
 	labelSelector := metav1.LabelSelector{MatchLabels: map[string]string{"app": constants.OSMControllerName}}
 	listOptions := metav1.ListOptions{
 		LabelSelector: labels.Set(labelSelector.MatchLabels).String(),
 	}
 	return deploymentsClient.List(context.TODO(), listOptions)
 }
+
+// getMeshNames returns a list of mesh names corresponding to meshes within the cluster
+func getMeshNames(clientSet kubernetes.Interface) []string {
+	var meshList []string
```

Comment:
If the meshLists are actual strong keys, I'd say to go for a set and have `meshExists` just do a lookup instead.
This is CLI and doesn't matter that much, but I'll probably try to be more insistent with this on the control plane; We have way too many loops over lists that can be indexed.

Reply:
Using a set makes sense here, will update.